drivers/hmis/app/graphql/types/hmis_schema/current_living_situation.rb

```diff
@@ -23,6 +23,7 @@ def self.configuration
     # TODO(2024) enable
     # hud_field :cls_subsidy_type, Types::HmisSchema::Enums::Hud::RentalSubsidyType
     hud_field :verified_by
+    hud_field 'CLSSubsidyType', HmisSchema::Enums::Hud::RentalSubsidyType
```

Comment:
We're going for consistency here to use snake case for all the fields
```suggestion
    hud_field :cls_subsidy_type, HmisSchema::Enums::Hud::RentalSubsidyType
```